src/parameterTypes.ts

```diff
@@ -0,0 +1,47 @@
+import { defineParameterType } from '@cucumber/cucumber';
+import { getTestDataFilePath } from '@utils';
+import fse from 'fs-extra';
```

Comment:
Will discuss if it possible to use built-in constants instead of file reading

Reply:
already discussed